PWGGA/GammaConvBase/AliCaloPhotonCuts.cxx

```diff
@@ -2167,8 +2167,19 @@ Bool_t AliCaloPhotonCuts::ClusterIsSelectedAODMC(AliAODMCParticle *particle,TClo
       if ( particle->Phi() < fMinPhiCut || particle->Phi() > fMaxPhiCut ) return kFALSE;
       if ( fClusterType == 3 && particle->Eta() < fMaxEtaInnerEdge && particle->Eta() > fMinEtaInnerEdge ) return kFALSE;
     }
-    if(particle->GetMother() > -1 && (static_cast<AliAODMCParticle*>(aodmcArray->At(particle->GetMother())))->GetPdgCode() == 22){
-        return kFALSE;// no photon as mothers!
+    // if(particle->GetMother() > -1 && (static_cast<AliAODMCParticle*>(aodmcArray->At(particle->GetMother())))->GetPdgCode() == 22){
+    //     //  cout << "kicking out photon as mother" << endl;
+    //     // printf(Form("UniqueID=%i PDG=%i  MotherID=%i MotherPDG=%i \n",particle->GetUniqueID(),particle->GetPdgCode(),particle->GetMother(),(static_cast<AliAODMCParticle*>(aodmcArray->At(particle->GetMother())))->GetPdgCode()));
+    //     return kFALSE;// no photon as mothers!
+    // }
+    
+    // reject photons with daughter photons instead, to end up only with final photon, not initial
```

Comment:
This implementation should be discussed, at least we all should agree on this @nschmidtALICE 
@FriederikeBock . Nevertheless, it should also be implemented for ESD at the same time

Reply:
implemented changes for ESD

Reply:
Change seems fine from my side. i'll approve the PR